Assignments/Program.cs

```diff
@@ -1 +1,86 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T4 - GCD & LCM Computer
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using static System.ConsoleColor;
+
+namespace Spark;
+
+public class T4 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "GCD & LCM Generator:-" + "\x1B[0m");
+      List<List<int>> testCases = new () { new () { 5, 10, 15, 21 }, new () { 6, 18, 36, 54, 96 }, 
+         new () { 6, 8, 12, 10, 30, 72, 134 }, new () { 2, 3, 5, 7, 11, 13 }, new () { 17, 34, 51, 68, 36 } };
+      for (int i = 0; i < testCases.Count; i++) {
+         Console.Write ($"\nTest case-{i + 1}: ");

```

Comment:
Don't use \n as line ending. See [here](https://www.cs.toronto.edu/~krueger/csc209h/tut/line-endings.html#:~:text=Text%20files%20created%20on%20DOS,(%22%5Cn%22).).

Reply:
Change in other places also.